common/gobuild.py

```diff
@@ -62,7 +62,7 @@ def build(parent, source_dir, target):
     if os.path.isdir("%s/main" % source_dir):
         source_dir += "/main"
     p = subprocess.Popen(
-        ["go", "build", "-i", "-ldflags=-s -w",  "-o", target],
```

Comment:
the `-i`  flag has been deprecated for a while; `go build 1.16` takes it a step further by removing it entirely